cypher/cypher-docs/src/docs/dev/ql/administration/alias/show-database-alias-columns.asciidoc

```diff
@@ -1,11 +1,12 @@
 [options="header" cols="2l,4,1,1,2"]
 |===
 |Column |Description|Default Output|Full Output| Notes
-|name|The name of the database alias.|{check-mark}|{check-mark}|
-|database|The names of the target database.|{check-mark}|{check-mark}|
+|name|The fully qualified name of the database alias.|{check-mark}|{check-mark}|
```

Comment:
it means that it might include the namespace/composite database the alias belong to, so given:
```
CREATE ALIAS composite.alias1 FOR ...
CREATE ALIAS alias2 FOR ...
```

`SHOW ALIASES YIELD name` return:
```
"composite.alias1"
"alias2"
```
and not just:
```
"alias1"
"alias2"
```
which would be the alias names.

Reply:
I see! Thanks for the explanation.